community-website/src/community-project-boilerplate-docgen/src/widgets/search-box.md

```diff
@@ -11,13 +11,13 @@ The Search Box component displays a search box that lets the user search for a s
 
 ## Options
 
-| Attribues         | Type      | Description
-| -                 | -         | -
-| `placeholder?`    | `string`  | The label of the input placeholder.
-| `submitTitle`     | `string`  | The submit button title.
-| `resetTitle`      | `string`  | The reset button title.
-| `searchAsYouType` | `boolean` | If you disable this option, new searches will only be triggered by clicking the search button or by pressing the enter key while the search box is focused.
-| `autofocus?       | `boolean` | Should the search box be focused on render?.
+| Attribues          | Type      | Description
```

Comment:
Attribues -> Name

Reply:
currently on all the other pages it's either
- Attribute: https://community.algolia.com/angular-instantsearch/widgets/breadcrumb.html
- Attributes: https://community.algolia.com/angular-instantsearch/widgets/sort-by.html
- Props: https://community.algolia.com/angular-instantsearch/widgets/toggle.-
- Attribures: https://community.algolia.com/angular-instantsearch/widgets/infinite-hits.html 
- Attribues: here

I think I would leave Attributes and just fix the typo. Putting `Name` in there would just introduce another inconsistency

Reply:
I'd change it everywhere to `Attribute` (since it's singular, not plural)